ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/DataMapperComponent.java

```diff
@@ -0,0 +1,206 @@
+package io.syndesis.qe.pages.integrations.detail;
+
+import static com.codeborne.selenide.CollectionCondition.size;
+import static com.codeborne.selenide.CollectionCondition.sizeGreaterThanOrEqual;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+
+import io.syndesis.qe.pages.SyndesisPageObject;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Created by sveres on 11/14/17.
+ */
+@Slf4j
+public class DataMapperComponent extends SyndesisPageObject {
+
+	private static final class Element {
+		public static final By ROOT = By.cssSelector("syndesis-data-mapper-host > div > data-mapper");
```

Comment:
Cant it be only By.cssSelector("data-mapper") it is basically selecting the same element.

Reply:
yes, it can!